tests/closetcoach_crawler/myntra_spider/spiders/test_product_page_urls.py

```diff
@@ -0,0 +1,66 @@
+"""
+This script is used to test the ProductPageUrlScraper spider from the myntra_spider
+project. It tests the start_requests and parse methods of the spider.
+"""
+
+import unittest
+from unittest.mock import MagicMock
+from scrapy.utils.project import get_project_settings
+from scrapy.utils.test import get_crawler
+from scrapy.utils import reactor
+from twisted.internet import asyncioreactor
+
+
+from closetcoach_crawler.myntra_spider.spiders.product_page_urls import (
+    ProductPageUrlScraper,
+)
+from tests.test_data.constants import TEST_PRODUCT_URLS
+
+# Install the AsyncioSelectorReactor and set it as the default reactor for Scrapy
```

Comment:
Do we need this to run the spider?, I don't think so

Reply:
Yeah, I will do the changes and update the code.

Reply:
@satishjasthi, I have updated the code please check it.